src/bubo-cache.cc

```diff
@@ -23,8 +23,8 @@ BuboCache::~BuboCache() {
     delete strings_table_;
 }
 
-void BuboCache::initialize(v8::Local<v8::Object> ignoredAttrs) {
-    ignored_attributes_.Reset(ignoredAttrs);
+void BuboCache::initialize(std::vector<std::string> ignoredAttrs) {
```

Comment:
In general I've always followed the convention of using const references when passing non-trivial types by value since it avoids having to copy the data structure twice. Obviously the performance impact is minimal here and it's possible that newer C++ compilers are smarter anyway, but still changing this argument to `const std::vector<std::string>& ignoredAttrs` would (I think) result in the same result and a more consistent style.